neural_compressor/pruner/pytorch_pruner/schedulers.py

```diff
@@ -59,7 +59,7 @@ def get_scheduler(config):
     return SCHEDULERS[name](config)
 
 
-class Scheduler:
+class BaseScheduler:
```

Comment:
shall we call it as PruningScheduler to avoid mix up with INC optimization orchestration API?